mrtarget/common/ElasticsearchQuery.py

```diff
@@ -40,8 +40,8 @@ def __init__(self, res):
 
 class ESQuery(object):
 
-    def __init__(self, es = None, dry_run = False):
-        self.handler = es if es else new_es_client()
+    def __init__(self, es, dry_run = False):
```

Comment:
@afaulconbridge is it better transparently passing the connections conf (all related about connections) and then let the class create the state and other object inside instead of passing others objects through? sharing addresses isn't good

Reply:
Yes it will be, but I need to do that in the harmonized configuration branch which will build upon this PR. Same thing goes for the Redis configuration too.